dev-tools/phive.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <phive xmlns="https://phar.io/phive">
-  <phar name="composer-normalize" version="^2.1.2" installed="2.1.2" location="./tools/composer-normalize" copy="false"/>
-  <phar name="composer-require-checker" version="^2.0" installed="2.0.0" location="./tools/composer-require-checker" copy="false"/>
-  <phar name="phpstan" version="^0.12" installed="0.12.9" location="./tools/phpstan" copy="true"/>
+  <phar name="composer-normalize" version="^2.2" installed="2.2.4" location="./tools/composer-normalize" copy="false"/>
+  <phar name="composer-require-checker" version="^2.1" installed="2.1.0" location="./tools/composer-require-checker" copy="false"/>
+  <phar name="phpstan" version="^0.12" installed="0.12.11" location="./tools/phpstan" copy="true"/>
```

Comment:
can we somehow drop `installed` lock?

Reply:
We should not: a new patch version of PHPStan can report new errors. Not locking the version would make our CI fail on a regular basis.

Reply:
0.12.12+ totally crashes, actually.